tests/unit/Runner/DefaultTestResultCacheTest.php

```diff
@@ -0,0 +1,34 @@
+<?php declare(strict_types=1);
+/*
+ * This file is part of PHPUnit.
+ *
+ * (c) Sebastian Bergmann <sebastian@phpunit.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+namespace PHPUnit\Runner;
+
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @covers \PHPUnit\Runner\DefaultTestResultCache
+ * @small
+ */
+final class DefaultTestResultCacheTest extends TestCase
+{
+    /**
+     * @var DefaultTestResultCache
+     */
+    private $subject;
+
+    protected function setUp(): void
+    {
+        $this->subject = new DefaultTestResultCache();
```

Comment:
I believe it's a matter of personal taste, but personally I would prefer not to set up the _system under test_ in the `setUp()` method, what do you think?